src/block-components/typography/style.js

```diff
@@ -0,0 +1,204 @@
+/**
+ * External dependencies
+ */
+import {
+	getFontFamily, clampInheritedStyle, useStyles, getStyles,
+} from '~stackable/util'
+import { Style as StyleComponent } from '~stackable/components'
+
+const getStyleParams = ( options = {} ) => {
+	const {
+		selector,
+		inherit,
+		inheritMin,
+		inheritMax,
+		hoverSelector,
+	} = options
+
+	return [
+		{
+			selector,
+			styleRule: 'fontSize',
+			attrName: 'fontSize',
+			hasUnits: 'px',
+			responsive: 'all',
+			hover: 'all',
+			hoverSelector,
+			valueCallback: ( _value, getAttribute, device, state ) => {
```

Comment:
You can use `clampCallback` instead to generate the clamped value based on the inherited value